GliaWidgets/SecureConversations/SecureConversations.WelcomeStyle.swift

```diff
@@ -3,40 +3,125 @@ import UIKit
 
 extension SecureConversations {
     /// Theme of the secure conversations welcome view.
-    public class WelcomeStyle {
+    public struct WelcomeStyle: Equatable {
         /// Style of the view's header (navigation bar area).
         public var header: HeaderStyle
 
         /// Title shown in the header.
-        public var title: String
+        public var headerTitle: String
 
         /// Title shown in the welcome area.
-        public var welcomeTitle: String
+        public var welcomeTitleStyle: TitleStyle
 
         /// Description showm in the welcome area.
-        public var welcomeDescription: String
+        public var welcomeSubtitleStyle: SubtitleStyle
+
+        public var checkMessagesButtonStyle: CheckMessagesButtonStyle
+
+        public var messageTitleStyle: MessageTitleStyle
+
+        public var messageTextViewStyle: MessageTextViewStyle
+
+        public var sendButtonStyle: SendButtonStyle
 
         /// View's background color.
         public var backgroundColor: UIColor
 
         /// - Parameters:
         ///   - header: Style of the view's header (navigation bar area).
-        ///   - title: Title shown in the header.
+        ///   - headerTitle: Title shown in the header.
         ///   - welcomeTitle: Title shown in the welcome area.
         ///   - welcomeDescription: Description showm in the welcome area.
         ///   - backgroundColor: View's background color.
         public init(
             header: HeaderStyle,
-            title: String,
-            welcomeTitle: String,
-            welcomeDescription: String,
+            headerTitle: String,
+            welcomeTitleStyle: TitleStyle,
+            welcomeSubtitleStyle: SubtitleStyle,
+            checkMessagesButtonStyle: CheckMessagesButtonStyle,
+            messageTitleStyle: MessageTitleStyle,
+            messageTextViewStyle: MessageTextViewStyle,
+            sendButtonStyle: SendButtonStyle,
             backgroundColor: UIColor
         ) {
             self.header = header
-            self.title = title
-            self.welcomeTitle = welcomeTitle
-            self.welcomeDescription = welcomeDescription
+            self.headerTitle = headerTitle
+            self.welcomeTitleStyle = welcomeTitleStyle
+            self.welcomeSubtitleStyle = welcomeSubtitleStyle
+            self.checkMessagesButtonStyle = checkMessagesButtonStyle
+            self.messageTitleStyle = messageTitleStyle
+            self.messageTextViewStyle = messageTextViewStyle
+            self.sendButtonStyle = sendButtonStyle
             self.backgroundColor = backgroundColor
         }
     }
 }
+
+extension SecureConversations.WelcomeStyle {
+    public struct TitleStyle: Equatable {
```

Comment:
And I believe these structs also need public documentation.

Reply:
Addressed in d13ff2b076b1fb2dd2347ac2679f10961f2a0b7b.